scripts/mea/et/internal_witness.py

```diff
@@ -272,14 +286,41 @@ def process_verifier_notes(self):
                 edge['note'] = self.process_comment(note)
             elif file_id in self._env_models_json and start_line in self._env_models_json[file_id]:
                 env = self._env_models_json[file_id][start_line]
-                comment = env["comment"]
+                comment = env.get("comment", None)
                 relevant = env.get("relevant", False)
+                name = env.get("name", None)
+                action_type = env.get("type", None)
+                self._logger.warning(f"type: {action_type} for name {name}")
+                if name is not None:
```

Comment:
Simplify to `if name`. But this is very bad stile, process simple comments separately